cmd/create.go

```diff
@@ -18,16 +18,32 @@ package main
 
 import (
 	"fmt"
+
 	"github.com/spf13/cobra"
 )
 
-func newDocsCmd() *cobra.Command {
-	return &cobra.Command{
-		Use:   "docs",
-		Short: "Generate API reference docs. Coming soon",
-		Long:  `Generate API reference docs. Coming soon`,
+func newCreateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Scaffold a Kubernetes API or webhook.",
+		Long:  `Scaffold a Kubernetes API or webhook.`,
 		Run: func(cmd *cobra.Command, args []string) {
 			fmt.Println("Coming soon.")
 		},
 	}
+	cmd.AddCommand(
+		newAPICommand(),
+	)
+
+	foundProject, version := getProjectVersion()
+	// It add webhook v2 command in the following 2 cases:
+	// - There are no PROJECT file found.
```

Comment:
What is the reasoning for this case ? IMO, absence of project file indicates an error and we can't assume anything about the project structure at that point.

Reply:
Here is more about what is the default commends to show.
The reason of this choice is that if a user is starting from scratch and trying to explore what commands and flags does kubebuilder support, the user should be able to see the new one.

This piece of code is about registering the command instead of running it. When running the command, we do required PROJECT to present.

Reply:
Got it. I miss-understood earlier. Thanks for clarification.